### DISCUSSION TITLE
Can it run on p40 GPU?Thinks

### DISCUSSION THREAD

**Discussion by YongZhang2023** [SYSTRAN/faster-whisper#344] [General]
Can it run on p40 GPU?Thinks

**Comment by guillaumekln** [comment]
Yes.

**Comment by dgoryeo** [comment]
Hi @YongZhang2023 , how do you like P40 for running Whipser? I was thinking about buying one but was not too sure about it yet. Any insights would be helpful. Thanks.

**Reply by neoyxm** [reply]
it's good enough for testing or studying purpose.  And 24G version is recommeded.

**Comment by alexellis** [comment]
@guillaumekln does the P40 still run without Cuda 12 support? I thought I saw you mention an issue with it being held back on Cuda 11?
